.travis.yml

```diff
@@ -2,4 +2,30 @@ language: java
 sudo: false
 cache:
   directories:
-      - $HOME/.m2/repository
+    - "$HOME/.m2/repository"
+
+install: mvn install -DskipTests=true -Dmaven.javadoc.skip=true
+script:
+  - jdk_switcher use oraclejdk8
+  - mvn test
+  - jdk_switcher use oraclejdk7
+  - mvn test
+before_deploy: bin/build-release.sh
+
+matrix:
+  fast_finish: true
+  include:
+    - env: STORM_RELEASE="0.10.0" MESOS_RELEASE="0.28.0"
+    - env: STORM_RELEASE="0.10.0" MESOS_RELEASE="0.27.2"
+    - env: STORM_RELEASE="0.9.6" MESOS_RELEASE="0.28.0"
+    - env: STORM_RELEASE="0.9.6" MESOS_RELEASE="0.27.2"
+
+# Deploy archives to Github release tags
+deploy:
+  provider: releases
+  api_key:
+    secure: DMwskItRMoVMpmYt3trBy4WlQN37ZnT28p1jYhP6xQOD1CfgZC+ggGiWr+aobfD89ZvztGdRPXyBCorsErqXEPLIA/dSAfJaOr20o6KzyFz284vw08dLwIvjev/eRYuT5BuRVCyFkt4MkTQEEGJdBE1Fb0nc8xdaylDR6CqthyI/qEE1dx4/yFoyBTBHQOBI3EynOG+h3D6XKQCfheNyX3i63St2wyI0hfTnanGNpsdxwgO5H4SW7p48/f+z4kjIt9IxHL7Lr8b56WbKvo97OqXzn+OFiCO8m/DtDD+HTUqjwDQA5oYvMx9bBHmZEyz/HgTnNsgH0RqTfo97AqezV37LSzJyWupF6Nk5N3k+UJu5ck/EvaiOBjDsye9sM1O6W5XICUfgNKHHvx6Ow5nVBY7gyrcCsJ+w65sda6Ij731GxmWEv2u9rfpq0/mNRvPwf/wFS9JN7SwNzsJPTYswsNyJoH7o/SVH7CiRavUTyCJJvelSimY9GwOWBCVNQIR8cioOncqH1mp1oxbSAVutcoppUt/jDNrxQwFM+XeI8AifK/CwoujSR6e3QKCPyRh3t5QICHwDcAh5qKDNGrLKEjTejCX+MdDKro+qzh4w7/T8jcZEKS+TpPC3cnFiT5N4joJ0EQ7oENRLpTR2JA4K38EPMSluma2dzveTXdz74QE=
```

Comment:
@erikdw this is actually the encrypted version of the api_key :). That's why we can commit it to the github repo, it's the recommended way from travis-ci.

After merging, one can update the deploy block by running : 
```
gem install travis
travis setup -f releases
```
It will ask you for github credentials in order to be able to create tokens. more info here https://docs.travis-ci.com/user/deployment/releases/